yaptide/routes/frontent_log_routes.py

```diff
@@ -0,0 +1,34 @@
+from flask_restful import Resource
+from flask import request
+
+from yaptide.routes.utils.response_templates import yaptide_response
+from yaptide.persistence.models import FrontendLogModel, UserModel
+from yaptide.persistence.database import db
+from yaptide.routes.utils.decorators import requires_auth
+
+
+class FrontendLogs(Resource):
+    """Receives logs from the forntent and saves them to the database"""
+
+    @staticmethod
+    @requires_auth()
+    def post(user: UserModel):
+        """Method saving frontend logs to database and returning status"""
+        print("POST /logs received")
+        json_data = request.get_json(force=True)
+
+        logs = json_data['logs']
+
+        for entry in logs:
+            log = FrontendLogModel(
+                user_id=user.id,
+                timestamp=entry['timestamp'],
+                level=entry['level'],
+                message=entry['message'],
+                browser=entry['browser'],
+                user_ip=entry['user_ip'],
+            )
+            db.session.add(log)
```

Comment:
I suggest to validate the `json_data` before getting some data from it:

Take a look on this approach
https://github.com/yaptide/yaptide/blob/7aa222f075b779c454f3a5a92cb350a8da4fa0e1/yaptide/routes/task_routes.py#L26-L31